app/routers/guest/applications.py

```diff
@@ -717,3 +718,41 @@ async def find_alternative_credit_option(
             borrower=new_application.borrower,
             award=new_application.award,
         )
+
+
+@router.get(
+    "/applications/uuid/{uuid}/access-external-onboarding",
+    tags=[util.Tags.applications],
+)
+async def access_external_onboarding(
+    session: Session = Depends(get_db),
+    application: models.Application = Depends(
+        dependencies.get_scoped_application_as_guest_via_uuid(statuses=(models.ApplicationStatus.SUBMITTED,))
+    ),
+) -> RedirectResponse:
+    """
+    Set the application borrower_accessed_external_onboarding_at and redirects to
+    lender.external_onboarding_system_url.
+
+    :param session
+    :param application
+    :return: A redirect to the lender.external_onboarding_system_url.
+    :raise: HTTPException if the application has a lender without an external_onboarding_system_url.
+    """
+    with rollback_on_error(session):
+        if not application.lender.external_onboarding_url:
+            raise HTTPException(
+                status_code=status.HTTP_409_CONFLICT,
```

Comment:
Per our docs: https://credere.readthedocs.io/en/latest/contributing/api.html#api-design
```suggestion
                status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
```